app/views/self_service/offices/_offices_table.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="table-wrapper">
+  <table class="offices-table">
+    <thead>
+      <tr>
+        <th class="offices-table__address">
+          <%= Office.human_attribute_name(:address) %>
+        </th>
+        <th class="offices-table__address-postcode">
+          <%= Office.human_attribute_name(:address_postcode) %>
+        </th>
+        <th class="offices-table__telephone-number">
+          <%= Office.human_attribute_name(:telephone_number) %>
+        </th>
+        <th class="offices-table__email-address">
+          <%= Office.human_attribute_name(:email_address) %>
+        </th>
+        <th class="offices-table__disabled-access">
+          <%= Office.human_attribute_name(:disabled_access) %>
+        </th>
+        <th class="offices-table__action"></th>
+      </tr>
+    </thead>
+    <tbody data-dough-filter-rows>
+      <% offices.each do |office| %>
+        <tr class="t-office-table-row">
+          <td class="t-address" data-dough-filterable><%= office_address_table_cell(office) %></td>
+          <td class="t-address-postcode" data-dough-filterable><%= office.address_postcode %></td>
+          <td class="t-telephone-number" data-dough-filterable><%= office.telephone_number %></td>
+          <td class="t-email-address" data-dough-filterable><%= office.email_address %></td>
+          <td class="t-disabled-access"><%= office.disabled_access ? 'Yes' : 'No' %></td>
+          <td>
+            <% if office == @firm.main_office %>
+              <span class="t-main-office">Main Office</span>
+            <% else %>
```

Comment:
And also, because this is a step on the way, it may not be obvious that the address etc. in the Firm records is going to be removed and migrated to become this first office record.